.github/workflows/ci.yml

```diff
@@ -29,9 +29,11 @@ jobs:
       - uses: actions/checkout@v3
       - uses: actions/setup-node@v3
         with:
-          node-version: 18
+          node-version: 18.15.0
       - uses: pnpm/action-setup@v2
         name: Install pnpm
+        with:
+          version: 8.3.0
```

Comment:
It is not necessary according to https://github.com/pnpm/action-setup: `Optional when there is a packageManager field in the package.json.`